apps/frontend/src/components/global/groups/GroupManagement.vue

```diff
@@ -80,15 +80,44 @@
         </v-chip>
       </template>
       <template #[`item.actions`]="{item}">
-        <div v-if="item.role === 'owner' || adminPanel">
+        <v-tooltip color="secondary" left>
+          <template #activator="{on}">
+            <v-icon small title="Info" data-cy="info" class="mr-2" v-on="on">
+              mdi-information
+            </v-icon>
+          </template>
+          <v-card color="transparent" flat max-width="600px" overflow-y-auto>
+            <span>
+              <v-card-title>{{ item.name }}</v-card-title>
+              <v-card-text>
+                Visibility: {{ item.public ? 'Public' : 'Private' }} <br />
```

Comment:
don't use breaks.  let v-row/v-col handle this

Reply:
Used divs instead - for some reason v-row doesn't adopt the padding of its container.